content/en/technical-debt.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Technical Debt
+status: Feedback Appreciated
+category: property
+---
+
+
+Technical debt is a concept in software development that reflects the implied cost of additional rework caused by choosing an easy solution for now instead of using a better approach that would take longer. It is a part of every software development project since trade-offs between speed, cost, and quality are impossible to avoid. Technical debts could arrive due to various defects in documentation, code, automation, design, architecture, etc. The best practices to counter technical debt include
```

Comment:
```suggestion
Technical debt is a concept in software development that reflects the implied cost of additional rework. Applications are continuously updated and improved, and today's technical decisions will impact tomorrow's changes. Caused by trade-offs between speed, cost, and quality, technical debt is impossible to avoid completely. Technical debts can result from errors in documentation, code, automation, design, architecture, etc. 
```

Reply:
Added a sentence as to why change is needed and removed "caused by choosing an easy over a better approach" as it sounds too judgmental. The rest are just stylistic changes.